contrib/platform/src/com/sun/jna/platform/win32/Oleacc.java

```diff
@@ -0,0 +1,205 @@
+    int ROLE_SYSTEM_PROPERTYPAGE = 0x26;
+    int ROLE_SYSTEM_INDICATOR = 0x27;
+    int ROLE_SYSTEM_GRAPHIC = 0x28;
+    int ROLE_SYSTEM_STATICTEXT = 0x29;
+    int ROLE_SYSTEM_TEXT = 0x2a;
+    int ROLE_SYSTEM_PUSHBUTTON = 0x2b;
+    int ROLE_SYSTEM_CHECKBUTTON = 0x2c;
+    int ROLE_SYSTEM_RADIOBUTTON = 0x2d;
+    int ROLE_SYSTEM_COMBOBOX = 0x2e;
+    int ROLE_SYSTEM_DROPLIST = 0x2f;
+    int ROLE_SYSTEM_PROGRESSBAR = 0x30;
+    int ROLE_SYSTEM_DIAL = 0x31;
+    int ROLE_SYSTEM_HOTKEYFIELD = 0x32;
+    int ROLE_SYSTEM_SLIDER = 0x33;
+    int ROLE_SYSTEM_SPINBUTTON = 0x34;
+    int ROLE_SYSTEM_DIAGRAM = 0x35;
+    int ROLE_SYSTEM_ANIMATION = 0x36;
+    int ROLE_SYSTEM_EQUATION = 0x37;
+    int ROLE_SYSTEM_BUTTONDROPDOWN = 0x38;
+    int ROLE_SYSTEM_BUTTONMENU = 0x39;
+    int ROLE_SYSTEM_BUTTONDROPDOWNGRID = 0x3a;
+    int ROLE_SYSTEM_WHITESPACE = 0x3b;
+    int ROLE_SYSTEM_PAGETABLIST = 0x3c;
+    int ROLE_SYSTEM_CLOCK = 0x3d;
+    int ROLE_SYSTEM_SPLITBUTTON = 0x3e;
+    int ROLE_SYSTEM_IPADDRESS = 0x3f;
+    int ROLE_SYSTEM_OUTLINEBUTTON = 0x40;
+
+    /**
+     * Retrieves the child ID or IDispatch of each child within an accessible container object.
+     *
+     * @see <a href="https://docs.microsoft.com/en-us/windows/win32/api/oleacc/nf-oleacc-accessiblechildren">AccessibleChildren function (oleacc.h)</a>
+     *
+     * @param paccContainer [in] Pointer to the container object's IAccessible interface.
+     * @param iChildStart [in] Specifies the zero-based index of the first child that is retrieved.
+     *                    This parameter is an index, not a child ID, and it is usually set to zero (0).
+     * @param cChildren [in] Specifies the number of children to retrieve. To retrieve the current number of children,
+     *                  an application calls IAccessible::get_accChildCount.
+     * @param rgvarChildren [out] Pointer to an array of VARIANT structures that receives information about the container's children.
+     *                      If the vt member of an array element is VT_I4, then the lVal member for that element is the child ID.
+     *                      If the vt member of an array element is VT_DISPATCH, then the pdispVal member for that element is
+     *                      the address of the child object's IDispatch interface.
+     * @param pcObtained [out] Address of a variable that receives the number of elements in the rgvarChildren array that is populated
+     *                   by the AccessibleChildren function. This value is the same as that of the cChildren parameter;
+     *                   however, if you request more children than exist, this value will be less than that of cChildren.
+     * @return the HRESULT. If successful, returns S_OK. If not successful, returns one of the error values, or another standard COM error code.
+     */
+    HRESULT AccessibleChildren(Pointer paccContainer, long iChildStart, long cChildren, VARIANT[] rgvarChildren, LongByReference pcObtained);
```

Comment:
Windows `LONG` is 32-bit, so these need to be mapped as `int` / `IntByReference`.  Lots of other instances, just noting this one.

Reply:
Sounds good, I've changed all mappings to use `int` / `IntByReference`.